config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_containerclusters.container.cnrm.cloud.google.com.yaml

```diff
@@ -193,6 +193,48 @@ spec:
                     required:
                     - disabled
                     type: object
+                  rayOperatorConfig:
```

Comment:
It's nice that a lot more fields are supported in this CRD, though we do need field coverage for spec in testing especially for big changes like this.

Considering that there are already breaking changes, I think you'll probably want to only focus on the updatable taint fields and not to add other new fields.